docs/static/style-guide/formats.md

```diff
@@ -0,0 +1,197 @@
+> This library provides various functions for formatting the data in the preferred style.
+
+## Formatting Functions
+
+#### `sszvis.formatText(x)`
+
+Any given data is returned as a string and will be formatted as Text.
+
+```table
+rows:
+  - Function: formatText
+    Example: sszvis.formatText(16000)
+    Result: "'16000'"
+  - Function:
+    Example: sszvis.formatText("Jahr")
+    Result: "'Jahr'"
+```
+
+#### `sszvis.formatNumber(x)`
+
+A number greater than 9'999 will have a thousands separator space.
+
+```table
+rows:
+  - Function: formatNumber
+    Example: sszvis.formatNumber(16000)
+    Result: "16 000"
+  - Function:
+    Example: sszvis.formatNumber(5000)
+    Result: "5000"
+```
+
+#### `sszvis.formatPreciseNumber(p, d)`
+
+`p` = The desired precision
+`d` = The number to be formatted
+
+A number will be formatted to a particular precision. It is a function that accepts numbers and returns formatted values. This function is "curried", meaning that it is a function with multiple arguments, but when you call it with less than the full number of arguments, it returns a function that takes less arguments and has the arguments you did provide "pre-filled" as parameters.
+
+Note that preciseNumber(2, 14.1234) is equivalent to preciseNumber(2)(14.1234)
+
+```table
+rows:
+  - Function: formatPreciseNumber
+    Example: sszvis.formatPreciseNumber(2, 14.1234)
+    Result: 14.12
+  - Function:
+    Example: sszvis.formatPreciseNumber(3, 14.1234)
+    Result: 14.123
+  - Function:
+    Example: sszvis.formatPreciseNumber(3)(14.1234)
```

Comment:
This is duplicated

Reply:
Yes, but on purpose to demonstrate both ways to write this function.